### DISCUSSION TITLE
Can't get into admin page

### DISCUSSION THREAD

**Discussion by Gooberpatrol66** [dani-garcia/vaultwarden#2431] [Q&A] [answered]
### Deployment environment

* vaultwarden version: 1.24.0

* Install method: OS package

* Clients used: 

* Reverse proxy and version: 

* MySQL/MariaDB or PostgreSQL version:  postgres 13.5

* Other relevant details:

### Steps to reproduce

set ADMIN_TOKEN in /etc/vaultwarden.env, restart vaultwarden service, go to vaultwarden/admin page, copy/paste token into field, press enter

### Expected behaviour

admin page loads

### Actual behaviour

the page to enter the token just reloads

**Comment by BlackDex** [comment]
A few things i think which could be the case. Either wrong token. Or some browser extension is blocking cookies. Or something is wrong with the DOMAIN config which could cause some weird issues.

Or... Something wrong with with the reverse proxy.

**Comment by Gooberpatrol66** [comment]
I don't have cookie blockers enabled to my knowledge.
`DOMAIN=https://localhost:8443`
These are my apache settings
```
        <Location /vaultwarden/>
                RewriteEngine On
                RewriteCond %{HTTP:Upgrade} =websocket [NC]
                RewriteRule /notifications/hub(.*) ws://localhost:3012/$1 [P,L]
                ProxyPass http://localhost:8000/

                ProxyPreserveHost On
                RequestHeader set X-Real-IP %{REMOTE_ADDR}s
        </Location>
```

**Comment by BlackDex** [comment, marked as answer]
Looks like your a using a sub-path but you do not have that configured in the DOMAIN variable as is pointed out in the wiki 

I suggest to read the proxy example `Apache in a sub-location (by ss89)` here https://github.com/dani-garcia/vaultwarden/wiki/Proxy-examples

**Reply by Gooberpatrol66** [reply]
That fixed it, thank you
